pkg/bgpv1/manager/reconcilerv2/service.go

```diff
@@ -147,7 +161,7 @@ func (r *ServiceReconciler) reconcileServices(ctx context.Context, p ReconcilePa
 
 		// get services to reconcile and to withdraw.
 		// Note : we should only call svc diff only once in a reconcile loop.
-		toReconcile, toWithdraw, err := r.diffReconciliationServiceList()
+		toReconcile, toWithdraw, err := r.diffReconciliationServiceList(p)
```

Comment:
In cases we call partial reconciliation before full reconciliation and init is not yet done, error will be returned.
Should we attempt to trigger full reconciliation in those cases? For both in v1 and v2 versions. 

I am thinking if there is a such a situation, we will not be able to recover unless something changes in BGP configuration which will trigger full reconciliation.

Reply:
I would say performing a partial reconciliation before full reconciliation is done is a bug in the reconciler, but yeah, to be more resilient we can put this recovery logic (catching this error running full reconcile upon it) into the reconcilers.

Reply:
This is no longer an issue - we now initialize the store in the new `Init(instance)` reconciler method.